packages/http/src/types.ts

```diff
@@ -15,7 +15,7 @@ export type TPrismHttpComponents<LoaderInput> = Partial<
 >;
 
 // TODO: should be complete | and in the @stoplight/types repo
-export type IHttpMethod = 'get' | 'put' | 'post' | 'delete' | 'options' | 'head' | 'patch' | 'trace'; // ... etc
+export type IHttpMethod = HttpMethod | 'trace';
```

Comment:
Wait why is trace not in there?

Reply:
It's in the `HttpExtendedMethod` type but it has some other verbs we do not care about.